mesonbuild/backend/ninjabackend.py

```diff
@@ -40,7 +39,7 @@
 from ..linkers import ArLinker, RSPFileSyntax
 from ..mesonlib import (
     File, LibType, MachineChoice, MesonException, OrderedSet, PerMachine,
-    ProgressBar, quote_arg
+    ProgressBar, quote_arg, cmd_quote, gcc_rsp_quote, get_rsp_threshold
```

Comment:
cmd_quote is not used any more here, right?

Reply:
No, it is still used when writing the ninja file